config/config.go

```diff
@@ -8,10 +8,10 @@ import (
 
 // App holds the config for the whole application.
 type App struct {
-	Debug        bool
-	AsyncAPIDoc  []byte      `split_words:"true"`
+	Debug        bool        `desc:"Enable or disable debug logs"`
+	AsyncAPIDoc  []byte      `split_words:"true" desc:"Path or URL to a valid AsyncAPI doc (v2.0.0 is only supported)"`
```

Comment:
```suggestion
	AsyncAPIDoc  []byte      `split_words:"true" desc:"Path or URL to a valid AsyncAPI doc (v2.0.0 is only supported)"`
```

Only `2.0.0` or `2.1.0` also? :)

Reply:
Unfortunately, 2.0.0 because go parser only parses up to that version.

Reply:
Ahhh... ok, I forgot about that.